src/backend/database.py

```diff
@@ -163,6 +163,17 @@ def init_database():
         },
         "max_participants": 16,
         "participants": ["william@mergington.edu", "jacob@mergington.edu"]
+    },
+    "Manga Maniacs": {
+        "description": "Explore the fantastic stories of the most interesting characters from Japanese Manga (graphic novels).",
```

Comment:
Please change this description to be inspired by Japanese Manga.
It needs more personality to attract students.